.github/workflows/ci.yml

```diff
@@ -97,6 +97,22 @@ jobs:
           git config --global user.name ${GITHUB_ACTOR}
       - name: Pytest check
         run: check/pytest --actually-quiet
+  pytest37:
+    name: Pytest Ubuntu
```

Comment:
Update the name of this and the 3.6 check so that they are differentiated from each other.

Reply:
Done.

Reply:
Let's be a little more explicit. Call this check "Pytest Ubuntu Python 3.7" and the other check "Pytest Ubuntu Python 3.6"

Reply:
Done.